nixos/modules/virtualisation/kvmgt.nix

```diff
@@ -50,11 +50,17 @@ in {
       nameValuePair "kvmgt-${name}" {
         description = "KVMGT VGPU ${name}";
         serviceConfig = {
-          Type = "oneshot";
+          Type = "forking";
           RemainAfterExit = true;
+          Restart = "on-failure";
+          RestartSec = 5;
           ExecStart = "${pkgs.runtimeShell} -c 'echo ${value.uuid} > /sys/bus/pci/devices/${cfg.device}/mdev_supported_types/${name}/create'";
```

Comment:
How about writing a udev rule instead that gets triggered when the device appears.

Reply:
It is also possible to make udev starting this service.

Reply:
@Mic92 This will not work. Device is created earlier than `/sys/bus/pci/devices/${cfg.device}/mdev_supported_types/` tree is populated.